### PR TITLE
Update hackathon page

### PR DESCRIPTION
**Pull request by mirpedrol**
- Add group leader names
- Add social activities information
- Add welcome and summary talks to schedule
- Add safety information from last hackathons

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1245"></a>

### CODE REVIEW

**Comment by ewels** [comment]
@nf-core-bot fix linting

File: markdown/events/2022/hackathon-october-2022.md [hunk 1]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
 
 You can add your community event using [this link](https://seqera.typeform.com/streaming-event).
 
+## Social Activities
+
+During the hackathon, we will have a few light-hearted fun and games!
+
+### On-site
+
+- Monday evening: Beers at Seqera Office!
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:250]:
```suggestion
- Monday evening: Beers and pizza (non-alcoholic options available) at the Seqera Office!
```

File: markdown/events/2022/hackathon-october-2022.md [hunk 2]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
 
 We will be monitoring the status of the pandemic as the event draws closer and reserve the right to make changes in our policies if we deem that to be necessary.
+
+# Safety Information at the Hackathon
+
+Please note that our [Code of Conduct](https://nf-co.re/code_of_conduct) applies to the Hackathon, and all participants need to abide by our guidelines to participate. We should all feel responsible for making nf-core events safe and fun for everyone.
+
+In addition, please respect the following at all times during the hackathon:
+
+- **Never share the jitsi and zoom links** for the hackathon on social media.
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:282]: Ooh this is an old one! 😅

File: markdown/events/2022/hackathon-october-2022.md [hunk 3]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media. Not respecting the latter amounts to doxing, which is also in direct violation of the nf-core [Code of Conduct](https://nf-co.re/code_of_conduct).
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:283]:
```suggestion
- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media.
```

File: markdown/events/2022/hackathon-october-2022.md [hunk 4]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+
+## Reporting CoC Violations during the hackathon
+
+In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and the time-zones they are located in are indicated.
+
+- Alexander Peltzer (CET)
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:289]: Should just be the 3x safety team. No mention of time zones, just say online / in person.

File: markdown/events/2022/hackathon-october-2022.md [hunk 5]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+- Gisela Gabernet (CET)
+- James A. Fellows Yates (CET)
+- Matthias Hörtenhuber (CET)
+- Maxime Garcia (CET)
+- Renuka Kudva (CET)
+- Kevin Menden (CET)
+- Friederike Hanssen (CET)
+- Michael Heuer (PST)
+
+You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officer, Renuka Kudva, will contact you to follow up on your report.
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:301]:
```suggestion
You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officers will contact you to follow up on your report.
```

File: markdown/events/2022/hackathon-october-2022.md [hunk 6]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+- Your contact information.
+- Identifying information (e.g. names, nicknames, pseudonyms) of the participant who has violated the Code of Conduct.
+- The behaviour that was in violation and the circumstances surrounding the incident.
+- The approximate time of the behaviour (if different than the time the report was made)
+- Other people involved in the incident if applicable.
+- If you believe the incident is ongoing, please let us know.
+- If there is a publicly available record (e.g. mailing list record, a screenshot - please attach if so).
+- Any additional helpful information.
+
+After you file a report, uur safety officer, Renuka Kudva, will contact you to follow up on your report.
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:314]:
```suggestion
```

File: markdown/events/2022/hackathon-october-2022.md [hunk 7]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+- Matthias Hörtenhuber (CET)
+- Maxime Garcia (CET)
+- Renuka Kudva (CET)
+- Kevin Menden (CET)
+- Friederike Hanssen (CET)
+- Michael Heuer (PST)
+
+You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officer, Renuka Kudva, will contact you to follow up on your report.
+
+In your email report, please do your best to include:
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:303]: Not sure that we need this level of detail tbh

File: markdown/events/2022/hackathon-october-2022.md [hunk 8]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+- The behaviour that was in violation and the circumstances surrounding the incident.
+- The approximate time of the behaviour (if different than the time the report was made)
+- Other people involved in the incident if applicable.
+- If you believe the incident is ongoing, please let us know.
+- If there is a publicly available record (e.g. mailing list record, a screenshot - please attach if so).
+- Any additional helpful information.
+
+After you file a report, uur safety officer, Renuka Kudva, will contact you to follow up on your report.
+
+### Who will read and handle reports?
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:316]: This stuff is all in the CoC already right? I think we can delete from here onwards..

File: markdown/events/2022/hackathon-october-2022.md [hunk 9]
```diff
@@ -14,15 +14,29 @@ location_name: Barcelona, Spain
 
 Please join us for the October 2022 nf-core hackathon!
 
-This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barcelona, Spain 🇪🇸. You can find out more information about the summit at <https://summit.nextflow.io>. Registration for the event will open in early July together with registration for the summit.
+This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barcelona, Spain 🇪🇸. You can find out more information about the summit at <https://summit.nextflow.io>.
 
 In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).
 
+Presentations will be run using Zoom and live-streamed to YouTube,
```
[the remark this answers is not in the document]
Comment (mirpedrol) [reply markdown/events/2022/hackathon-october-2022.md:21]: I took that from past hackathons. Will they happen only in gather town and won't be recorded to post on YouTube after the hackathon?

File: markdown/events/2022/hackathon-october-2022.md [hunk 10]
```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+
+### On-site
+
+- Monday evening: Beers at Seqera Office!
+
+- Tuesday evening: Dinner!
+
+- The traditional sock hunting in Gathertown will also be held in person! Take pictures of as many socks as you can find.
+
+- Finally, on Thursday, we will be running a short quiz!
```
[the remark this answers is not in the document]
Comment (mirpedrol) [reply markdown/events/2022/hackathon-october-2022.md:256]: Definitely not Thursday, sorry. We didn't say a day, should we say Wednesday?

File: markdown/events/2022/hackathon-october-2022.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ewels) [reply markdown/events/2022/hackathon-october-2022.md]: Oh I'm not sure now.. 🤔 Maybe needs discussion, can take it in Slack.

File: markdown/events/2022/hackathon-october-2022.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (ewels) [reply markdown/events/2022/hackathon-october-2022.md]: Wednesday will be a little short is the only thing, as we're stopping at lunch time.

File: markdown/events/2022/hackathon-october-2022.md [hunk 11]
```diff
@@ -142,9 +226,49 @@ Keep an eye out for others hosting community events in your area. We will be sha
...
+
+In addition, please respect the following at all times during the hackathon:
+
+- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media.
+
+## Reporting CoC Violations during the hackathon
+
+In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and their location (online/onsite) are indicated.
+
+- Chris Hakkaart (onsite)
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:270]:
```suggestion
- Cris Tuñí (onsite)
```

File: markdown/events/2022/hackathon-october-2022.md [hunk 12]
```diff
@@ -44,10 +84,18 @@ We will coordinate our work and the issues we are working on using a single GitH
 
 This group will focus on all kind of documentation, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. We are also planning on making introduction videos for each pipeline.
 
+  - [<i class="fab fa-slack"></i> `#hackathon-oct-2022-documentation`](https://nfcore.slack.com/archives/C041QU9SX09)
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:87]: Are these indented for a reason? It's breaking prettier linting..

File: markdown/events/2022/hackathon-october-2022.md (hunk 11 above)
Comment (mirpedrol) [reply markdown/events/2022/hackathon-october-2022.md:270]: oops 😇

File: markdown/events/2022/hackathon-october-2022.md (hunk 12 above)
Comment (mirpedrol) [reply markdown/events/2022/hackathon-october-2022.md:87]: They pass prettier linting in my computer 🤔 but I will remove the indentation just in case

**Review by mashehu — APPROVED** [review]

**Review by apeltzer — APPROVED** [review]

**Pull request merged**